socorro/lib/search_common.py

```diff
@@ -59,20 +61,20 @@
 HISTOGRAM_QUERY_TYPES = ("date", "number")
 
 
+@dataclass
 class SearchParam:
-    def __init__(self, name, value, operator=None, data_type=None, operator_not=False):
-        self.name = name
-        self.value = value
-        self.operator = operator
-        self.data_type = data_type
-        self.operator_not = operator_not
+    name: str
+    value: str
+    operator: Optional[str] = None
+    data_type: Optional[str] = None
+    operator_not: bool = False
```

Comment:
Switching this to dataclasses makes this code much easier to debug because it includes `__repr__` and `__str__` methods.